vendor/assets/javascripts/eui-1.0.0/eui.js

```diff
@@ -124,9 +124,11 @@
   // FUNCTIONALITY: #Accordion
 
     // Basic Accordion Functionality
-    $(".eui-accordion__header").click(function() {
-      $(this).siblings(".eui-accordion__body").slideToggle("fast");
-      $(this).closest(".eui-accordion").toggleClass("is-closed");
+    $(".eui-accordion__header").click(function(e) {
+      if (!$(e.target).is('i.eui-fa-info-circle')) {
```

Comment:
We should try to fix the issue without going into the vendor library (even if it is EUI). Did you look into doing that?

Reply:
yes, this is the fix.

Reply:
I'll take another pass at it, and let you know if I can come up with anything else